dist/src/main/java/io/camunda/application/commons/backup/BackupPriorityConfiguration.java

```diff
@@ -153,6 +154,7 @@ public static BackupPriorities getBackupPriorities(
             new DecisionRequirementsIndex(indexPrefix, isElasticsearch),
             new MetricIndex(indexPrefix, isElasticsearch),
             new ProcessIndex(indexPrefix, isElasticsearch),
+            new CorrelatedMessageIndex(indexPrefix, isElasticsearch),
```

Comment:
❓ Is there a specific reason for priority 5 ?

Reply:
I can also ask in the project channel and ping the data layer team.

Reply:
OK, let's try to clarify it in the channel then 👍

Reply:
https://camunda.slack.com/archives/C08MRKHJ0CD/p1756894396865369